apps/content/management/commands/seed_data.py

```diff
@@ -139,7 +121,7 @@ def seed_data(self):
         seeder.add_entity(
             SongPlayLog,
             random.randint(7777, 9999),
-            {"date_played": lambda x: random_date()}
+            {"date_played": lambda x: seeder.faker.past_datetime(start_date="-180d", tzinfo=pytz.UTC)},
```

Comment:
I modified the seed command to use faker instead of a custom function. Once less thing to maintain!